Bottom: 7fedb00fda6dae6e1ca942255c34fb580838fb9e
Top:    98f574c982230e5fc411b432bcc77c94513641d1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-20 16:05:01 -0500

Refresh of builder-add-new-host

---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index 67202894..eb6bccf7 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -3,10 +3,10 @@ daedalus.novatech-llc.com
 
 [docker-hosts]
 daedalus.novatech-llc.com
-buildbot-worker.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
 
-[buildbot-worker]
-buildbot-worker.novatech-llc.com
+[builder]
+builder.novatech-llc.com
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index 6c319766..3da6bb1f 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -3,10 +3,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
-buildbot-worker.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
 
-[buildbot-worker]
-buildbot-worker.novatech-llc.com
+[builder]
+builder.novatech-llc.com
